tests/smoke.rs

```diff
@@ -289,9 +291,9 @@ fn sp_smoke_test() {
                         })
             });
 
-            let sp = frame.sp() as usize;
-            eprintln!("sp  = {:p}", sp as *const u8);
-            if sp == 0 {
+            let sp = frame.sp();
+            eprintln!("sp  = {sp:p}");
+            if sp as usize == 0 {
```

Comment:
...why didn't I just write `is_null()` lol

Reply:
eugh, I really shouldn't have reviewed that day even with a dose of caffeine